packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -209,11 +211,11 @@ export default class ResolverImpl implements RuntimeResolver<Owner>, CompileTime
   }
 
   lookupBuiltInHelper(name: string): HelperDefinitionState | null {
-    return BUILTIN_KEYWORD_HELPERS[name] ?? null;
+    return (BUILTIN_KEYWORD_HELPERS as Record<string, object>)[name] ?? null;
   }
 
   lookupModifier(name: string, owner: Owner): Option<ModifierDefinitionState> {
-    let builtin = BUILTIN_MODIFIERS[name];
+    let builtin = (BUILTIN_MODIFIERS as Record<string, object>)[name];
```

Comment:
Can we (or do we) supply the same type definition such that it works correctly without the cast for *this one* here and below?